config/api.yml

```diff
@@ -461,6 +461,9 @@
         :identifier: miq_report_run
     :subcollections:
     - :results
+    :collection_actions:
+      :post:
+      - :name: import
```

Comment:
@abellotti looks like I was having the same issue here as I was having for tags/classifications. Just updated the api.yml file and this works as expected now